lib/linguist/languages.yml

```diff
@@ -5690,6 +5690,7 @@ Xojo:
   type: programming
   extensions:
   - ".xojo_code"
+  - ".xojo_image"
```

Comment:
Using the search query [`extension:xojo_image`](https://github.com/search?q=extension%3Axojo_image+NOT+nothack&type=Code), I downloaded the 567 existing `.xojo_image` files. They come from 49 repositories by 33 users. As per our contribution guidelines, this is well below our requirements on in-the-wild usage. I'd recommend using overrides until usage is high enough to add this extension in Linguist.

Reply:
No problem. I've removed the extension and example file.

Reply:
Is there any reason to think usage may grow rapidly in the next few months (ex. new extension change, new open source release, etc.)?

Reply:
Not specifically. It's fairly common to find it in Xojo projects. It also doesn't contain "code" but rather resource (image) references/specifications. So it's not as important as the other file types and doesn't benefit as significantly from highlighting.

Here is an example:
 [Plain](https://gitlab.com/xojo/BubblePopper/raw/master/App%20Icon.xojo_image)  
 [Highlighted](https://github-lightshow.herokuapp.com/?utf8=✓&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fjimmckay%2FXojoSyntaxTM%2Fmaster%2FXojo.tmLanguage&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgitlab.com%2Fxojo%2FBubblePopper%2Fraw%2Fmaster%2FApp%2520Icon.xojo_image&code=)

Reply:
Ok. Thanks for the explanation.